dashboard/app/views/pd/application/teacher1819_application_mailer/accepted.html.haml

```diff
@@ -0,0 +1,77 @@
+%p
+  Hello
+  = @application.first_name + ","
+
+%p
+  Congratulations! Code.org
+  - if @application.regional_partner_id
+    and
+    = @application.regional_partner.name
+    are
+  - else
+    is
+  excited to offer you a space in Code.org's 2018-19 Professional Learning
+  Program for
+  = @application.course + "!"
+  Please read below for details, and then make sure to register using the link
+  below within two weeks of receiving this message.
+
+%p
+  As a reminder, all participants in the Professional Learning Program are expected to:
+
+%ul
+  %li
+    Attend their assigned summer workshop
+  - if @application.regional_partner_id
+    %li
+      Attend their assigned academic year workshops
+  %li
+    Teach this curriculum in the 2018-19 school year
+
+%p
+  Here are your important next steps:
+%ul
+  %li
+    %strong
+      Within two weeks
+    of receiving this message,
+    %strong
+      please complete this form
+    to let us know whether or not you are accepting your seat in the program. If
+    you are accepting your seat, this form will also be your registration for
+    TeacherCon, your five-day summer workshop.
```

Comment:
Do we say TeacherCon even for local summer workshops? Or is this just a placeholder until we have the specific workshop data in teacher application? Same below.

Reply:
This is just a placeholder; note that the teacher acceptance emails are currently disabled for that very reason